packages/expo-updates/ios/EXUpdates/AppLoader/RemoteAppLoader.swift

```diff
@@ -101,19 +101,30 @@ public final class RemoteAppLoader: AppLoader {
           )
           return
         }
+        self.database.databaseQueue.async {
+          let embeddedUpdate = EmbeddedAppLoader.embeddedManifest(withConfig: self.config, database: self.database)
+          let extraHeaders = FileDownloader.extraHeadersForRemoteAssetRequest(
```

Comment:
same as my comment for android. it's not ideal to re-compute the extra header for each asset downloading. maybe we can compute once and pass the computed result to this function.

that would also save the overhead for `asset queue -> database queue -> asset queue` switching for download each asset.

Reply:
Good point, updated